packages/client/MessagePickupRepositoryClient.ts

```diff
@@ -0,0 +1,316 @@
+import { Client } from 'rpc-websockets'
+import { Logger } from '@nestjs/common'
```

Comment:
It still depends on @nestjs/common. It would be nice to use a simpler logger library as we do in other projects, in order to not make the client dependant on Nest.

Reply:
Resolved, add loglevel package